### ISSUE TITLE
Extra page for the good RMS and the FSF have done?

### ISSUE THREAD

**Issue by ossimalvin** [rms-support-letter/rms-support-letter.github.io#2441]
Since the other repo has an extra page detailing why they believe RMS is evil incarnate, I think there should be an extra page on this repo detailing what the FSF has done for open source and maybe some links to donation pages and other ways to support the project.

**Comment by shenlebantongying** [comment]
- https://jorgemorais.gitlab.io/justice-for-rms/
- https://sterling-archermedes.github.io
- https://www.wetheweb.org/post/cancel-we-the-web

**Comment by 6r1d** [comment]
[Proof RMS is Still Important](https://open.lbry.com/@Lunduke:e/RMSStillImportant:6?r=GAaJ9s3EZNYqp61oaAXUcUzgEMPmhhf1)

[Why RMS is a Great Mascot and spokesperson for Free Software
](https://open.lbry.com/@Lunduke:e/RMSGoodMascot:b?r=GAaJ9s3EZNYqp61oaAXUcUzgEMPmhhf1)

**Comment by nukeop** [comment]
Please document this, it's very important.

**Comment by dodobyte** [comment]
that's a good one
https://geoff.greer.fm/2019/09/30/in-defense-of-richard-stallman/

**Comment by Maciejfiedler** [comment]
> Since the other repo has an extra page detailing why they believe RMS is evil incarnate, I think there should be an extra page on this repo detailing what the FSF has done for open source and maybe some links to donation pages and other ways to support the project.

Yeah this guy literally changed the whole goddamn world with GNU, GNU/Linux and FSF.

**Comment by michaelkarlcoleman** [comment]
RMS has arguably done more to make the world a better place than any other computer programmer, living or dead.

**Comment by jobbautista9** [comment]
We can put the extras to the repo's wiki, so that it can be easily edited.

**Comment by shenlebantongying** [comment]
I think a Redditor has done a big job
https://www.reddit.com/r/freesoftware/comments/meot29/dissecting_hate_speech_the_rms_open_letter/

**Comment by romanrm** [comment]
See https://stallmansupport.org/ and specifically https://stallmansupport.org/richard-stallman-honors-and-awards.html

**Comment by shenlebantongying** [comment]
https://stallmansupport.org/ has been endorsed by RMS himself and that page link to us.

![image](https://user-images.githubusercontent.com/20123683/114263380-1f676e80-99b3-11eb-84b6-dddbb4ecf5dc.png)

![image](https://user-images.githubusercontent.com/20123683/114263416-4e7de000-99b3-11eb-9cc0-b4051583f557.png)
